lib/foreman_snapshot_management/engine.rb

```diff
@@ -63,7 +63,7 @@ class Engine < ::Rails::Engine
           context.add_pagelet :main_tabs,
                               :name => N_('Snapshots'),
                               :partial => 'hosts/snapshots_tab',
-                              :onlyif => proc { |host| host&.compute_resource&.capable?(:snapshots) }
+                              :onlyif => proc { |host| host&.compute_resource&.capable?(:snapshots) && host&.uuid }
```

Comment:
I would put the host.uuid before the compute_resource check and do it like:
`host.uuid.present?`

Reply:
Applied